bench/wtperf/runners/many-dhandle-stress.wtperf

```diff
@@ -0,0 +1,21 @@
+# This workload uses 10000 tables and the workload is evenly distributed among them. 
+# Configured with idle_table_cycle where it creates, opens and drops a table and it
+# fails fail if that takes more than configured time. 
```

Comment:
"fails fail" --> "fails"

Reply:
For your consideration, rewording as follows:
"The workload creates, opens and drop tables, and it fails if the time taken is more than the configured idle_table_cycle"

Reply:
Updated the comment section.